mooncake-store/conf/master.yaml

```diff
@@ -0,0 +1,20 @@
+enable_gc: false
+enable_metric_reporting: true
+metrics_port: 9003
+rpc_port: 50051
+rpc_thread_num: 4
+rpc_address: "0.0.0.0"
+rpc_conn_timeout_seconds: 0
+rpc_enable_tcp_no_delay: true
+
+default_kv_lease_ttl: 200
```

Comment:
The default_kv_least_ttl is changed to 5000 ( mooncake::DEFAULT_DEFAULT_KV_LEASE_TTL) in a recent commit. Would you mind updating this value accordingly?